### DISCUSSION TITLE
ActionIconGroup - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#32] [Q&A]
# ActionIconGroup - Lobe UI

ActionIconGroup is a component used to render multi buttons

https://ui.lobehub.com/components/action-icon-group

**Comment by rdmclin2** [comment]
可以加个 size 控制大小吗？

**Comment by rdmclin2** [comment]
dropdownMenuk 不配置时应该显示为空？
